assets/sass/components/_footer.scss

```diff
@@ -0,0 +1,163 @@
+.footer {
+  width: 100%;
+&__wrapper {
+  margin: 0;
+  padding: 0;
+  border:0;
+  background-color: #333333;
+  width: 100%;
+ }
+&__adress {
+  padding-bottom: 50px;
+  margin:0;
+  padding-top: 100px;
+  font-size: 14px;
+    width: 100%;
+}
+&__adress p {
+  padding-bottom: 10px;
+}
+&__adress p:first-child{
+  font-size: 16px;
+  padding-bottom: 22px;
+}
+&__adress p:last-child{
+  padding-bottom: 0;
+}
+p {
+  margin-left: 38px;
+  color: #FFFFFF;
+  font-family: Montserrat, SemiBold;
+  font-size: 16px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+&__email{
+  padding-bottom: 50px;
+}
+&__email p:first-child{
+  font-size: 14px;
+  margin-top:0;
+  padding-bottom: 21px;
+  margin-bottom: 0;
+}
+&__phone{
+padding-bottom: 50px;
+margin-top: 0;
+width: 100%;
+}
+&__phone p:first-child{
+  box-sizing: border-box;
+  padding-bottom: 22px;
+}
+&__connect{
+  font-size: 14px;
+  padding-bottom: 100px;
+  padding-top: 0;
+  width: 100%;
+  margin:0;
+  margin-top:0;
+  &__wrapper{
+  margin-left: 38px;
+  }
+  &__wrapper img:nth-child(3) {
+      margin-left: 50px;
+      margin-top:20px;
+      margin-right: 50px;
+      margin-bottom: 0;
+    }
+    &__wrapper p:first-child{
+      margin-left: 0px;
+    }
+  }
+}
+.footer-copyrights {
+  padding-top: 41px;
+  padding-bottom: 41px;
+  background-color: #222222;
+  margin:0;
+  width: 100%;
+&__wrapper p{
+  overflow: hidden;
+  color: #999999;
+}
+&__wrapper p:first-child{
+  overflow: hidden;
+  margin-bottom: 20px;
+}
+}
+ @include tablet()
+ {
+.footer{
+  p{
+  margin-left: 82px;
+  }
+ &__connect__wrapper{
+  margin-left: 82px;
+ }
+}
+.footer-copyrights__wrapper
+{
+  display: inline;
+}
+.footer-copyrights__wrapper p{
+  display: inline;
+}
+.footer-copyrights__wrapper p:first-child {
+  float: right;
+  margin-right: 80px;
+}
+.footer-copyrights__wrapper p:last-child{
+    float: left;
+    margin-right: 80px;
+  }
+
+}
+@include desktop-xlg()
+{
+  .footer{
+  .footer__wrapper{
```

Comment:
this is wrong! .footer { &__wrapper {...} } is correct. Please check nesting one more time. Pull from master and check my code, that is also part of learning. ;)